### PR TITLE
Dockerfile + Deploy - AT THE ESTABLISHED TIME!!

### PR DESCRIPTION
**Pull request by Tomenos18**
In this PR, I include one Dockerfile, one script to create the corresponding image of the container and the documentation about the creation and deploy the container of the web application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,28 @@ Some ideas for obtaining a :gift: if you are the first that:
 User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml)
+[Tomás Pelayo Benedet](https://github.com/Tomenos18/lab1-git-race) |779691 | [![Build Status](https://github.com/Tomenos18/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/Tomenos18/lab1-git-race/actions/workflows/ci.yml)
 your name | your nia | your Travis-ci status
+
```
Comment (fjlopez) [inline README.md:19]: Move the changes to the file *description.md*

File: README.md [hunk 2]
```diff
@@ -14,4 +14,28 @@ Some ideas for obtaining a :gift: if you are the first that:
...
+## How to deploy with Dockerfile
+
+It's very simple, just follow the following steps:
+
+1. Run the script called "create_image.sh" from the "scripts" folder. This if going to create the image that we need:
+
+```
```
Comment (fjlopez) [inline README.md:26]: use: 
````
```console
$ ls
message
```
````

Renders as:
```console
$ ls
message
```

File: description.md [hunk 3]
```diff
@@ -1 +1,22 @@
-Complete me!
+## How to deploy with Dockerfile
+
+It's very simple, just follow the following steps:
+
+1. Run the script called "create_image.sh" from the "scripts" folder. This if going to create the image that we need:
+
+```
```
Comment (fjlopez) [inline description.md:7]: Same, use:
````
```console
$ ls
message
```
````
Renders as:
```console
$ ls
message
```

**Comment by fjlopez** [comment]
Some changes requested. Don't close this PR.

**Comment by Tomenos18** [comment]
Done all requested changes

**Comment by Tomenos18** [comment]
Conflicts also solved

File: README.md (hunk 1 above)
Comment (Tomenos18) [reply README.md:25]: Done

File: README.md (hunk 2 above)
Comment (Tomenos18) [reply README.md]: Done

File: description.md (hunk 3 above)
Comment (Tomenos18) [reply description.md]: Done

**Comment by Tomenos18** [comment]
@fjlopez All done

**Comment by fjlopez** [comment]
Sorry for the mess. You are awarded with a :gift:.

**Pull request merged**